src/rsp_restspawner/spawner.py

```diff
@@ -42,9 +42,10 @@ class LabStatus(str, Enum):
     of the lab controller.
     """
 
-    STARTING = "starting"
+    PENDING = "pending"
     RUNNING = "running"
```

Comment:
This would all be so much nicer if k8s did this cleanly rather than Pending/Running/Succeeded/Failed/Unknown and then also having to parse the Reason field to figure out how it got there.